utils/storecli.go

```diff
@@ -69,8 +69,8 @@ func NewFakeStoreCLI(r io.Reader) (*StoreCLI, error) {
 }
 
 // Executes nvme list, parses the output and returns a slice of model.StorageControllers
```

Comment:
`StorageControllers returns a slice of model.StorageControllers from the output of nvme list`